cmds/contest/main.go

```diff
@@ -1,8 +1,6 @@
-// Copyright (c) Facebook, Inc. and its affiliates.
-//
-// This source code is licensed under the MIT license found in the
-// LICENSE file in the root directory of this source tree.
```

Comment:
Won't the bot get angry about the missing copyright header?

Reply:
it will, good catch

Reply:
done